Tests/EmbeddedScripts.CSharp.MonoEvaluator.Tests/MonoCSharpRunnerTests.cs

```diff
@@ -0,0 +1,226 @@
+using System;
+using System.Threading.Tasks;
+using EmbeddedScripts.Shared.Exceptions;
+using HelperObjects;
+using Xunit;
+
+namespace EmbeddedScripts.CSharp.MonoEvaluator.Tests
+{
+    public class MonoCSharpRunnerTests 
```

Comment:
Tests are also your library usage examples. So it would be nice to add some examples that you know shouldn't work here. 
For example, raw function declaration will probably work using roslyn but will fail using mono.